.github/workflows/build-windows.yml

```diff
@@ -87,10 +87,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Setup msbuild
```

Comment:
I think some of these changes are the reason why the CI isn't running on this PR?

Reply:
The CI is not running because I am new to this repo. (I didn't commit any changes). And you need activity approve the ci after each change I made. (Very annoying for me and for you).

Reply:
I don't see any CIs in the Checks tab of this PR. Are you sure it's something related to admin privileges?

Reply:
![image](https://github.com/borglab/gtsam/assets/7043539/4a9f341d-a705-4c20-9a80-2e71f7928f14)

The review require, mean some one need to approve my CI. It already done here   
https://github.com/borglab/gtsam/pull/1413 with @dellaert.

Soon I will close this PR, and open a new one better results.

Reply:
I have the privilege, since I approved the CI for #1559. But I don't see the button appearing for this one...

Reply:
By closing and opening you approve it indirectly. 
But I don't have the privilege to show you, which button to push. I just now it somewhere in the red section in the printscreen. You should approve the review, and not the PR.
I want to close the PR, and open a new one.